website/docs/podman/podman-remote.md

```diff
@@ -40,10 +43,25 @@ $ ssh-keygen -t ed25519 -f ~/.ssh/id_ed25519
 
 Your public SSH key needs to be copied to the `~/.ssh/authorized_keys` file on the Linux server:
 
+<Tabs groupId="operating-systems">
+
+<TabItem value="win" label="Windows">
+
+```shell-session
+type ~\.ssh\id_ed25519.pub | ssh user@my-server-ip "cat >> .ssh/authorized_keys"
```

Comment:
We need to add a command prompt symbol before the actual command, as we used $ in the case of Linux and macOS.